server/monitor.go

```diff
@@ -2772,21 +2783,23 @@ func (s *Server) HandleHealthz(w http.ResponseWriter, r *http.Request) {
 						if !cc.isStreamCurrent(accName, sname) {
 							health.Status = "unavailable"
 							health.Error = fmt.Sprintf("JetStream stream %q for account %q is not current", sname, accName)
-							w.WriteHeader(http.StatusServiceUnavailable)
 							js.mu.RUnlock()
 							break Err
 						}
+						// Now do consumers.
+						for _, o := range stream.getConsumers() {
```

Comment:
I wonder if there was not a reason you did not add consumers to the probe when you did the original PR... but I don't recall.

Reply:
In any case was an oversight, good catch by @matthiashanel